parlai/core/hugging_face_bpe_helper.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from parlai.core.opt import Opt
+from typing import List
+
+
+class HuggingFaceBpeHelper(object):
+    @staticmethod
+    def add_cmdline_args(argparser):
+        parser = argparser.add_argument_group('HuggingFaceBpeHelper Arguments')
+        parser.add_argument(
+            '--bpe-vocab', type=str, help='path to pre-trained tokenizer vocab'
```

Comment:
should these arguments have a default? (like the one you put in the zoo?)

Reply:
Actually not, hmm, the purpose of having this is for loading fairseq / pretrained tokenizers, and they should be given each time. There is no universal one here.

Reply:
Actually... hold up. let's talk about this more, i just realized something

Reply:
Resolved offline.